src/components/modern/Icon.js

```diff
@@ -0,0 +1,188 @@
+import IconExchange from 'assets/icons/svg/icon-24-exchange.svg';
+import IconFailed from 'assets/icons/svg/icon-24-failed.svg';
+import IconGift from 'assets/icons/svg/icon-24-gift.svg';
+import IconHighFees from 'assets/icons/svg/icon-24-high-fees.svg';
+import IconImage from 'assets/icons/svg/icon-24-image.svg';
+import IconInfo from 'assets/icons/svg/icon-24-info.svg';
+import IconInvestment from 'assets/icons/svg/icon-24-investment.svg';
+import IconKey from 'assets/icons/svg/icon-24-key.svg';
+import IconLifebuoy from 'assets/icons/svg/icon-24-lifebuoy.svg';
+import IconLiquidityPool from 'assets/icons/svg/icon-24-liquidity-pool.svg';
+import IconLogout from 'assets/icons/svg/icon-24-logout.svg';
+import IconMail from 'assets/icons/svg/icon-24-mail.svg';
+import IconMenu from 'assets/icons/svg/icon-24-menu.svg';
+import IconMinus from 'assets/icons/svg/icon-24-minus.svg';
+import IconMute from 'assets/icons/svg/icon-24-mute.svg';
+import IconNote from 'assets/icons/svg/icon-24-note.svg';
+import IconPending from 'assets/icons/svg/icon-24-pending.svg';
+import IconPercentCircle from 'assets/icons/svg/icon-24-percent-circle.svg';
+import IconPhone from 'assets/icons/svg/icon-24-phone.svg';
+import IconPlus from 'assets/icons/svg/icon-24-plus.svg';
+import IconPower from 'assets/icons/svg/icon-24-power.svg';
+import IconQrCode from 'assets/icons/svg/icon-24-qrcode.svg';
+import IconQuestion from 'assets/icons/svg/icon-24-question.svg';
+import IconRecovery from 'assets/icons/svg/icon-24-recovery.svg';
+import IconRefresh from 'assets/icons/svg/icon-24-refresh.svg';
+import IconSearch from 'assets/icons/svg/icon-24-search.svg';
+import IconSelect from 'assets/icons/svg/icon-24-select.svg';
+import IconSendDown from 'assets/icons/svg/icon-24-send-down.svg';
+import IconSend from 'assets/icons/svg/icon-24-send.svg';
+import IconSettings from 'assets/icons/svg/icon-24-settings.svg';
+import IconSetttlement from 'assets/icons/svg/icon-24-settlement.svg';
+import IconSort from 'assets/icons/svg/icon-24-sort-ascending.svg';
+import IconSynthetic from 'assets/icons/svg/icon-24-synthetic.svg';
+import IconThumbUp from 'assets/icons/svg/icon-24-thumb-up.svg';
+import IconUser from 'assets/icons/svg/icon-24-user.svg';
+import IconWallet from 'assets/icons/svg/icon-24-wallet.svg';
+import IconWarning from 'assets/icons/svg/icon-24-warning.svg';
+
+// Assets: services icons
+import IconEthereum from 'assets/icons/svg/icon-24-ethereum.svg';
+import IconBinance from 'assets/icons/svg/icon-24-binance.svg';
+import IconXdai from 'assets/icons/svg/icon-24-xdai.svg';
+import IconPolygon from 'assets/icons/svg/icon-24-polygon.svg';
+import IconWalletConnect from 'assets/icons/svg/icon-24-wallet-connect.svg';
+
+const components = {
+  // Generic icons
+  'add-cash': IconAddCash,
+  'add-contact': IconAddContact,
+  'arrow-right': IconArrowRight,
+  'arrow-down': IconArrowDown,
+  'arrow-left': IconArrowLeft,
+  'arrow-up': IconArrowUp,
+  bullet: IconBullet,
```

Comment:
why are some keys wrapped in `' '`while others aren't?

Reply:
legal JS names can be used without quotes when defining an object, however for keys that are not legal JS identifiers you need to put them into quotes.